assembly/src/docker/build.sh

```diff
@@ -100,7 +100,7 @@ elif [ -n "${FROM_LOCAL}" ]; then
   if [ -n "${ARCHIVE}" ]; then
      DIST_DIR=${ARCHIVE}
   else
-     DIST_DIR="target/apache-activemq-*.tar.gz"
+     DIST_DIR="../../target/apache-activemq-*.tar.gz"
```

Comment:
Any reason to change the path here ?

Reply:
Good question, since I was not sure from where to launch the script because I did not find any doc about it, I may have changed it incorrectly. I will revert it asap.

Reply:
Fixed thx